deploy/base/deployment.yaml

```diff
@@ -101,6 +101,11 @@ spec:
                 configMapKeyRef:
                   name: store-configmap
                   key: bigtable.version
+            - name: BOUNCE
```

Comment:
what does this do? is it possible to add a comment here?

Reply:
Done